source-boilerplate/boilerplate.go

```diff
@@ -21,17 +22,27 @@ import (
 	"google.golang.org/grpc/metadata"
 )
 
+const LISTEN_TIMEOUT_SECONDS = 30
+
 // RunMain is the boilerplate main function of a capture connector.
 func RunMain(srv pc.DriverServer) {
 	var logConfig = &logConfig{}
 	var parser = flags.NewParser(logConfig, flags.Default)
 	var ctx, _ = signal.NotifyContext(context.Background(), syscall.SIGTERM, syscall.SIGINT)
 
 	var conn net.Conn
-	if server, err := net.Listen("tcp", ":2222"); err != nil {
+	server, err := net.ListenTCP("tcp4", &net.TCPAddr{
+		IP:   net.IPv4(127, 0, 0, 1),
```

Comment:
Likewise, is there a reason sources listen on `127.0.0.1` vs materializations on `0.0.0.0`? It seems like the external-reachability constraints should be the same for both.